pkg/authz/objecttype/model.go

```diff
@@ -29,7 +30,7 @@ type Model interface {
 	GetCreatedAt() time.Time
 	GetUpdatedAt() time.Time
 	GetDeletedAt() *time.Time
-	ToObjectTypeSpec() (*ObjectTypeSpec, error)
+	ToObjectTypeSpec(token *wookie.Token) (*ObjectTypeSpec, error)
```

Comment:
Can we add the Token to the `Model` itself and just suppress it (`-`) in the db field? Feels kinda hacky to change the signature of this method.